courses/migrations/0002_auto_20151120_1503.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='EdxAuthor',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('edx_uid', models.UUIDField(default=uuid.uuid4, help_text=b'Unique ID generated by the edx-platform', unique=True)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='course',
+            name='instructors',
+            field=models.ManyToManyField(to='courses.EdxAuthor'),
+        ),
+    ]
```

Comment:
Please rename this migration to something meaningful describing the action done: something like
`0002_new_model_edxauthor.py`